nativeApp/src/open_views/screens/Login/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { Button as B } from 'react-native-elements';
+
+import Input from '../../../lib/ui/forms/input';
+import { ColoursEnum } from '../../../lib/ui/colours';
+
+const Page = styled.View`
+  flex: 1;
+  justifyContent: space-around;
+  flexDirection: column;
+  alignItems: center;
+`;
+
+const Container = styled.View`
+  flex: 2;
+  justifyContent: center;
+`;
+
+const BottomContainer = styled(Container)`
+  flex: 1;
+  justifyContent: flex-end;
+  paddingBottom: 20;
+`;
+
+const Link = styled.Text`
+  textAlign: center;
+  color: #007bff;
+  fontSize: 17;
+`;
+
+
+const Image = styled.Image`
+  width: 100;
+  height: 100;
+`;
+
+const Submit = styled(B).attrs({
+  buttonStyle: {
+    backgroundColor: ColoursEnum.purple,
+    padding: 15,
+  },
+  containerStyle:{
+    width: 290,
+    paddingBottom: 10,
+  }
+})``;
+
+const Login = (props) => {
+
+  return (
+    <Page>
+
+      <Container>
+        <Image source={require('../../../../assets/images/logo_image.png')}/>
```

Comment:
all imports should be at the top of the page

Reply:
This is RN syntax for images.

Reply:
This is just asking webpack to load an image into memory as a data URL. It doesn't have anything to do with RN.